content-types/content-type-read-receipt/README.md

```diff
@@ -1,17 +1,12 @@
 # Read receipt content type
 
-![Status](https://img.shields.io/badge/Content_type_status-Standards--track-yellow) ![Status](https://img.shields.io/badge/Reference_implementation_status-Alpha-orange)
-
 This package provides an XMTP content type to support read receipts to messages.
 
-> **Important**  
+> [!IMPORTANT]
 > This standards-track content type is in **Alpha** status as this implementation doesn't work efficiently with the current protocol architecture. This inefficiency will be addressed in a future protocol release.
```

Comment:
@rygine - I was asked to remove this callout from the docs - so I did. Might we want to remove it here as well?

Reply:
no problem, will do!